backend/capellacollab/sessions/hooks/persistent_workspace.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import typing as t
+
+from capellacollab.core import models as core_models
+from capellacollab.sessions import operators
+from capellacollab.sessions.operators import models as operators_models
+from capellacollab.users import models as users_models
+
+from . import interface
+
+
+class PersistentWorkspacEnvironment(t.TypedDict):
+    pass
+
+
+class PersistentWorkspaceHook(interface.HookRegistration):
+    """Takes care of the persistent workspace of a user.
+
+    Is responsible for mounting the persistent workspace into persistent sessions.
+    """
+
+    def configuration_hook(
+        self,
+        operator: operators.KubernetesOperator,
+        user: users_models.DatabaseUser,
+        **kwargs,
+    ) -> tuple[
```

Comment:
This is a simplification that I put on top of the integrations. The advantage is to have less unused parameters in functions.

It works because we're only using named arguments when calling the hook. Unused arguments are catched by `kwargs`.